appeals/web/src/server/lib/dates.js

```diff
@@ -1,196 +1,213 @@
-import { formatInTimeZone } from 'date-fns-tz';
+import { formatInTimeZone, zonedTimeToUtc } from 'date-fns-tz';
 import enGB from 'date-fns/locale/en-GB/index.js';
-import logger from '#lib/logger.js';
-import { isValid, isBefore, isAfter, startOfDay, parseISO } from 'date-fns';
+import { isValid, isBefore, isAfter } from 'date-fns';
+import { padNumberWithZero } from '#lib/string-utilities.js';
 
 export const timeZone = 'Europe/London';
```

Comment:
`packages/appeals/constants/dates.js` includes this constant now, could be imported.

Reply:
done